cmd/infra/aws/create.go

```diff
@@ -596,13 +672,60 @@ yum install -y squid
 # By default, squid only allows connect on port 443
 sed -E 's/(^http_access deny CONNECT.*)/#\1/' -i /etc/squid/squid.conf
 systemctl enable --now squid
+
 mkdir -p /home/ec2-user/.ssh
 chmod 0700 /home/ec2-user/.ssh
 echo -e '%s' >/home/ec2-user/.ssh/authorized_keys
 chmod 0600 /home/ec2-user/.ssh/authorized_keys
 chown -R ec2-user:ec2-user /home/ec2-user/.ssh
 `
 
+const secureProxyConfigurationScript = `#!/bin/bash
```

Comment:
maybe we can extract these two script constants and the generation function in a separate file for better visibility given how significant they are?